doc/nrf/libraries/nrf_rpc/nrf_rpc_ipc.rst

```diff
@@ -0,0 +1,35 @@
+.. _nrf_rpc_ipc_readme:
+
+nRF RPC IPC Service transport
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF RPC IPC Service transport allows you to use the :ref:`nrf_rpc` library to execute procedures on remote processors.
+It uses the IPC Service no-copy mechanism.
+
+This feature is used in the :ref:`ble_rpc` library and also in the :ref:`nrf_rpc_entropy_nrf53` sample.
+
+Limitations
+***********
+
+1. The current implementation of the IPC service with the RPMSG backed allows to allocate fixed size buffer.
```

Comment:
```suggestion
1. The current implementation of the IPC service with the RPMSG backend allows to allocate a fixed size buffer.
```
- Should we open the abbreviation RPMSG?
- Use bullet points instead of numbers, as these are not sequential steps.

Reply:
I find a link to RPMsg so I replaced this sentence in all places with link.
Sure I switched to bullet point list